src/icons/BurnerWalletDivider.tsx

```diff
@@ -0,0 +1,224 @@
+            values="0 0 0 0 0.996078 0 0 0 0 0.847059 0 0 0 0 0.137255 0 0 0 1 0"
+          />
+          <feBlend
+            mode="normal"
+            in2="BackgroundImageFix"
+            result="effect1_dropShadow_57_21794"
+          />
+          <feBlend
+            mode="normal"
+            in="SourceGraphic"
+            in2="effect1_dropShadow_57_21794"
+            result="shape"
+          />
+        </filter>
+        <filter
+          id="filter1_d_57_21794"
+          x="263.75"
+          y="10"
+          width="45.5"
+          height="20"
+          filterUnits="userSpaceOnUse"
+          color-interpolation-filters="sRGB"
+        >
+          <feFlood flood-opacity="0" result="BackgroundImageFix" />
+          <feColorMatrix
+            in="SourceAlpha"
+            type="matrix"
+            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+            result="hardAlpha"
+          />
+          <feOffset />
+          <feGaussianBlur stdDeviation="2" />
+          <feComposite in2="hardAlpha" operator="out" />
+          <feColorMatrix
+            type="matrix"
+            values="0 0 0 0 1 0 0 0 0 0.483333 0 0 0 0 0.929029 0 0 0 1 0"
+          />
+          <feBlend
+            mode="normal"
+            in2="BackgroundImageFix"
+            result="effect1_dropShadow_57_21794"
+          />
+          <feBlend
+            mode="normal"
+            in="SourceGraphic"
+            in2="effect1_dropShadow_57_21794"
+            result="shape"
+          />
+        </filter>
+        <filter
+          id="filter2_d_57_21794"
+          x="12.25"
+          y="1"
+          width="34"
+          height="20"
+          filterUnits="userSpaceOnUse"
+          color-interpolation-filters="sRGB"
+        >
+          <feFlood flood-opacity="0" result="BackgroundImageFix" />
+          <feColorMatrix
+            in="SourceAlpha"
+            type="matrix"
+            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+            result="hardAlpha"
+          />
+          <feOffset />
+          <feGaussianBlur stdDeviation="2" />
+          <feComposite in2="hardAlpha" operator="out" />
+          <feColorMatrix
+            type="matrix"
+            values="0 0 0 0 0.996078 0 0 0 0 0.847059 0 0 0 0 0.137255 0 0 0 1 0"
+          />
+          <feBlend
+            mode="normal"
+            in2="BackgroundImageFix"
+            result="effect1_dropShadow_57_21794"
+          />
+          <feBlend
+            mode="normal"
+            in="SourceGraphic"
+            in2="effect1_dropShadow_57_21794"
+            result="shape"
+          />
+        </filter>
+        <filter
+          id="filter3_d_57_21794"
+          x="0.75"
+          y="11"
+          width="45.5"
+          height="20"
+          filterUnits="userSpaceOnUse"
+          color-interpolation-filters="sRGB"
+        >
+          <feFlood flood-opacity="0" result="BackgroundImageFix" />
+          <feColorMatrix
```

Comment:
Not sure if it will work in Safari, can you check it out?
![image](https://user-images.githubusercontent.com/49658988/189419137-ccc5b2ae-daf2-4fe6-a78a-40a0b8310f73.png)

Reply:
Works fine in the Safari browser
<img width="701" alt="Снимок экрана 2022-09-12 в 13 29 05" src="https://user-images.githubusercontent.com/5114069/189632077-d6f0d311-b764-4dbe-be48-ece3b6994617.png">